src/hooks/useRovingRowRef.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from '@linaria/core';
+
+import { cell } from '../style';
+import { useFocusRef } from './useFocusRef';
+
+const rowSelected = css`
+  outline: none;
+
+  &::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
Should check other cases where we can simplify this.
```suggestion
    inset: 0;
```